src/ontology/uberon-edit.obo

```diff
@@ -224207,6 +224207,43 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: located_in UBERON:0002371 ! bone marrow
 property_value: dcterms-date "2023-06-26T13:48:30Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480071
+name: skin of lobe of tail
+def: "A zone of skin that is part of the lobe tail." [https://orcid.org/0000-0002-0819-0473]
+synonym: "fluke skin" NARROW [PMID:21959845]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0013131 ! lobe of tail
+property_value: dcterms-date "2023-08-29T09:37:23Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480072
+name: skin of shin
+def: "A zone of skin on the front hindlimb zeugopod." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
```

Comment:
Consider adding logical definition 'zone of skin' and ('part of' some 'front hindlimb zeugopod')

Reply:
@aleixpuigb Agree with that. We need this term to describe the cell line SCCT10 (CVCL_VJ56) originated in PMID:24662767 (Supp Data Table 4  "left shin").